### PR TITLE
chore: website performance

### PR DESCRIPTION
**Pull request by Geminii**
The goal of this PR is to increase website performance on the loading of modules : 

Respect of purpose by @Atinux :  
- Use https://github.com/maoberlehner/vue-lazy-hydration
- List only the first 12 modules and lazy render more on scroll

Resolves #48

### CODE REVIEW

**Comment by Atinux** [comment]
When looking at the preview, there is an issue on the card background: https://modules-myg6xyl3z.vercel.app/

**Comment by Geminii** [comment]
> When looking at the preview, there is an issue on the card background: https://modules-myg6xyl3z.vercel.app/

In fact, i remove it by inadvertence sorry for this :) 
It could be fine now 👍

File: components/CardModule.vue [hunk 1]
```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="flex flex-col h-full">
+    <div class="relative flex flex-1 flex-col space-y-2 px-6 py-8 group">
+      <a :href="module.website" :aria-label="module.website" target="_blank" rel="noopener" class="absolute inset-0" />
+      <div class="transition-opacity duration-200 ease-in-out opacity-0 group-hover:opacity-100 absolute top-4 right-6 cursor-pointer">
+        <img alt="website" src="~/assets/icons/ext.svg" width="24" height="24">
+      </div>
+
+      <nuxt-image
```
Comment (pi0) [inline components/CardModule.vue:9]: I think svg is now supported by image module. /cc @farnabaz

**Review by pi0 — COMMENTED** [review]
🔥

File: components/CardModule.vue (hunk 1 above)
Comment (Geminii) [reply components/CardModule.vue:9]: We can integrate it with this [PR](https://github.com/nuxt/modules/pull/45) no ?

Comment (pi0) [reply components/CardModule.vue:9]: Yeah we may simply upgrade in this PR :) Since otherwise one will conflict

Comment (Geminii) [reply components/CardModule.vue:9]: I integrated it so you can closed #45 
Maybe in another PR, we could manage all images with `nuxt-image` directly ?

Comment (pi0) [reply components/CardModule.vue:9]: Thanks man! For rest of images i would suggest waiting for feature `nuxt-img` landing it is specially for migrating all `img` tags :)

**Comment by Geminii** [comment]
@Atinux could you review this one ?

**Pull request merged**